resources/js/src/app/store/modules/singleItem/ItemModule.js

```diff
@@ -77,6 +77,22 @@ const mutations =
         setVariationMarkInvalidProps(state, markFields)
         {
             state.variationMarkInvalidProperties = !!markFields;
+        },
+
+        setVariationPropertySurcharges(state, basePrice )
+        {
+            if (state.variation.documents[0].data.properties)
+            {
+                for (const property of state.variation.documents[0].data.properties)
+                {
+                    const percentage = property.percentage || property.property.percentage;
+
+                    if (!isNullOrUndefined(percentage))
```

Comment:
die Berechnung mit Percentage darf nur passieren, wenn am Artikel für das Merkmal kein eigenständiger Preis angegeben ist. In diesem Fall wird der angegebene Preis als Festpreis (nicht prozentual) genommen.